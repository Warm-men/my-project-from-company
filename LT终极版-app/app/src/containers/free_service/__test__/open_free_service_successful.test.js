import React from 'react'
import render from 'react-test-renderer'
import OpenFreeServiceSuccessfulContainer from '../refunding_free_service'
describe('open freeService successful page', () => {
  it('renders correctly', () => {
    expect(
      render.create(<OpenFreeServiceSuccessfulContainer />).toJSON()
    ).toMatchSnapshot()
  })
})
