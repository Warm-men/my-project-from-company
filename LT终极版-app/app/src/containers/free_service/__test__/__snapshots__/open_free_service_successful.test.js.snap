// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`open freeService successful page renders correctly 1`] = `
<View
  onLayout={[Function]}
  pointerEvents="box-none"
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "paddingBottom": 0,
      "paddingLeft": 0,
      "paddingRight": 0,
      "paddingTop": 20,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "white",
          "flexDirection": "row",
          "height": 44,
          "justifyContent": "space-between",
          "zIndex": 2,
        },
        Object {
          "borderBottomColor": "#f6f6f6",
          "borderBottomWidth": 1,
        },
        undefined,
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "height": "100%",
            "justifyContent": "center",
          },
          Object {
            "width": "15%",
          },
          undefined,
        ]
      }
    >
      <View
        accessible={true}
        isTVSelectable={true}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "height": 44,
            "justifyContent": "center",
            "opacity": 1,
            "paddingLeft": 20,
            "width": 50,
          }
        }
      >
        <Text
          style={
            Object {
              "display": "none",
            }
          }
        >
          返回按钮
        </Text>
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "#242424",
                "fontSize": 18,
              },
              undefined,
              Object {
                "fontFamily": "simple-line-icons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
        >
          
        </Text>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "width": "70%",
          },
          Object {
            "height": "100%",
            "justifyContent": "center",
          },
          undefined,
        ]
      }
    >
      <Text
        numberOfLines={1}
        style={
          Array [
            Object {
              "color": "#000",
              "fontSize": 17,
              "fontWeight": "400",
              "letterSpacing": -0.41,
              "textAlign": "center",
            },
            undefined,
          ]
        }
      >
        自在选
      </Text>
    </View>
    <View
      style={
        Array [
          Object {
            "height": "100%",
            "justifyContent": "center",
          },
          Object {
            "width": "15%",
          },
          undefined,
        ]
      }
    />
  </View>
  <View
    style={
      Array [
        Object {
          "height": 76,
          "marginTop": 88,
          "width": 76,
        },
        Object {
          "overflow": "hidden",
        },
      ]
    }
  >
    <Text
      style={
        Object {
          "display": "none",
        }
      }
    >
      图片
    </Text>
    <Image
      defaultSource={null}
      source={1}
      style={
        Object {
          "height": 76,
          "width": 76,
        }
      }
    />
  </View>
  <Text
    style={
      Object {
        "color": "#989898",
        "fontSize": 14,
        "marginTop": 20,
      }
    }
  >
    已关闭自在选
  </Text>
  <Text
    style={
      Object {
        "color": "#333333",
        "fontSize": 16,
        "marginTop": 14,
      }
    }
  >
    
  </Text>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "flex-end",
        "width": "100%",
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#EA5C39",
          "borderRadius": 4,
          "height": 44,
          "justifyContent": "center",
          "marginBottom": 10,
          "opacity": 1,
          "width": "90%",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FFFFFF",
            "fontSize": 14,
          }
        }
      >
        我知道了
      </Text>
    </View>
  </View>
</View>
`;
